src/Api/ItemList.php

```diff
@@ -0,0 +1,79 @@
+<?php
+
+namespace OpsWay\ZohoBooks\Api;
+
+
+class ItemList implements \ArrayAccess, \IteratorAggregate
```

Comment:
A description telling why this class is required should be nice. It could be placed alongside a short example and a link to ZohoBooks related docs.